website/content/docs/plugins/creation/index.mdx

```diff
@@ -212,89 +167,64 @@ Here's what you need to create releases using GitHub Actions:
+By opening an integration request, you are asking a member of the to Packer team to review your plugin integration configuration, plugin documentation, 
+and, finally, to open an internal pull-request to finalize the integration setup. 
 
-This is done by adding the block below for the respective plugin to the file [website/data/plugins-manifest.json](https://github.com/hashicorp/packer/blob/main/website/data/plugins-manifest.json).
-
-```json
-{
-  "title": "Scaffolding",
-  "path": "scaffolding",
-  "repo": "hashicorp/packer-plugin-scaffolding",
-  "version": "latest",
-  "sourceBranch": "main"
-}
-```
-
-If a plugin maintainer wishes to only include a specific version of released docs, then the `"version"` key in the above configuration should be set to a released version of the plugin. Otherwise it should be set to `"latest"`.
-
-The `"sourceBranch"` key in the above configuration ensures potential contributors can link back to source files in the plugin repository from the Packer docs site. If a `"sourceBranch"` value is not present, it will default to `"main"`.
-
-### Testing Plugin Documentation
-
-Before publishing the `docs.zip` file, you might want to preview your documentation changes.
-We provide a mechanism that allows to preview how the docs will look like within
-the Packer documentation.
-
-Follow the next steps to get the Packer website running and preview the documentation changes:
-
-- Get the [Packer source code](https://github.com/hashicorp/packer). Our website code is under the [website folder](https://github.com/hashicorp/packer/tree/main/website).
-- Generate the `docs.zip` file. You can find above the steps to do so.
-- Add the `zipFile` attribute to the plugin entry in `plugins-manifest.json`. The value should be the full path of the `docs.zip` generated. For example:
-
-```json
-{
-  "title": "Scaffolding",
-  "path": "scaffolding",
-  "repo": "hashicorp/packer-plugin-scaffolding",
-  "version": "latest",
-  "sourceBranch": "main",
-  "zipFile": "/Users/myuser/Packer/plugins/packer-plugin-scaffolding/docs.zip"
-}
-```
+Plugin integrations will be listed as a [Packer Integration](https://developer.hashicorp.com/packer/integrations), with details on how to install and use your the plugin. 
 
-- Go to the [website folder](https://github.com/hashicorp/packer/tree/main/website).
-  In the website README, follow the steps to [run the website with node](https://github.com/hashicorp/packer/tree/main/website#with-node).
-- Once the website is up and running, the plugin documentation should be available in `http://localhost:3000/docs`.
+Plugin integrations, once deployed, can be updated manually, or automatically upon a new release, by the plugin authors. Changes to the defined documentation structure 
+or parent repository should be communicated to the Packer team to ensure a working integration pipeline. 
```

Comment:
how should the communication be? through which channel?

Reply:
🤷‍♂️ - good question. I'll suggest opening an issue or sending an email to the packer integration distro.

Reply:
Maybe add the information here as well